Formula/adr-tools.rb

```diff
@@ -0,0 +1,21 @@
+class AdrTools < Formula
+  desc "CLI tool for working with Architecture Decision Records."
+  homepage "https://github.com/npryce/adr-tools"
+  url "https://github.com/npryce/adr-tools/archive/2.1.0.tar.gz"
+  sha256 "1ef028cfeaa1b262a5c62845aa8965be169705370983f9ff73b17ec77bf75f70"
+
+  def install
+    # Override adr-config with custom config to point to homebrew dirs
+    system "echo '#!/bin/bash' > src/adr-config"
+    system "echo 'echo adr_bin_dir=\"#{bin}\"' >> src/adr-config"
+    system "echo 'echo adr_template_dir=\"#{prefix}\"' >> src/adr-config"
```

Comment:
Use `Pathname#write` instead:
```ruby
(buildpath/"src/adr-config").write <<-EOS.undent
  your content
  goes here
EOS
```

Please ask upstream how we could specify this config without overriding their file.